.github/workflows/cli.yml

```diff
@@ -13,6 +13,18 @@ jobs:
   build:
     runs-on: ${{ matrix.os }}
 
+    services:
+      postgres:
+        image: postgres:10.8
```

Comment:
Why this one specifically? There are less precise (and newer?) ones too right

Reply:
Wow you are fast 😎 
Mostly created this PR to get the action runs. But feel free to raise things to improve ;)
Here - yes, definitely can go for newer and less precise version